spec/intl.html

```diff
@@ -1267,281 +1267,273 @@ <h1>Temporal.Calendar.prototype.dateUntil ( _one_, _two_, _options_ )</h1>
             1. Return 𝔽(_eraYear_).
           </emu-alg>
         </emu-clause>
 
         <emu-clause id="sup-temporal.calendar.prototype.year">
-          <h1>Temporal.Calendar.prototype.year ( _dateOrDateTime_ )</h1>
+          <h1>Temporal.Calendar.prototype.year ( _temporalDateLike_ )</h1>
           <p>This definition supersedes the definition provided in <emu-xref href="#sec-temporal.calendar.prototype.year"></emu-xref>.</p>
           <p>
-            The `year` method takes one argument _dateOrDateTime_.
+            The `year` method takes one argument _temporalDateLike_.
             The following steps are taken:
           </p>
           <emu-alg>
             1. Let _calendar_ be the *this* value.
             1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
+            1. If Type(_temporalDateLike_) is not Object or _temporalDateLike_ does not have an [[InitializedTemporalDate]] or [[InitializedTemporalYearMonth]] internal slot, then
+              1. Set _temporalDateLike_ to ? ToTemporalDate(_temporalDateLike_).
             1. If _calendar_.[[Identifier]] is *"iso8601"*, then
-              1. Let _year_ be ? ISOYear(_dateOrDateTime_).
+              1. Let _year_ be ! ISOYear(_temporalDateLike_).
             1. Else,
-              1. If Type(_dateOrDateTime_) is not Object or _dateOrDateTime_ does not have an [[ISOYear]] internal slot, then
-                1. Set _dateOrDateTime_ to ? ToTemporalDate(_dateOrDateTime_).
-              1. Let _year_ be the result of implementation-defined processing of _dateOrDateTime_ and the value of _calendar_.[[Identifier]].
+              1. Let _year_ be the result of implementation-defined processing of _temporalDateLike_ and the value of _calendar_.[[Identifier]].
             1. Return 𝔽(_year_).
           </emu-alg>
         </emu-clause>
 
         <emu-clause id="sup-temporal.calendar.prototype.month">
-          <h1>Temporal.Calendar.prototype.month ( _dateOrDateTime_ )</h1>
+          <h1>Temporal.Calendar.prototype.month ( _temporalDateLike_ )</h1>
           <p>This definition supersedes the definition provided in <emu-xref href="#sec-temporal.calendar.prototype.month"></emu-xref>.</p>
           <p>
-            The `month` method takes one argument _dateOrDateTime_.
+            The `month` method takes one argument _temporalDateLike_.
             The following steps are taken:
           </p>
           <emu-alg>
             1. Let _calendar_ be the *this* value.
             1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
-            1. If Type(_dateOrDateTime_) is Object and _dateOrDateTime_ has an [[InitializedTemporalMonthDay]] internal slot, then
+            1. If Type(_temporalDateLike_) is Object and _temporalDateLike_ has an [[InitializedTemporalMonthDay]] internal slot, then
```

Comment:
Doesn't match polyfill - polyfill throws the exception after performing several Get operations I think, because it falls through to ToTemporalDate. If `.month` added to prototype of PlainMonthDay is allowed behavior, then we should change this; otherwise, we can change ToTemporalDate to TypeError on receiving a PMD. WDYT?

Reply:
This was a pre-existing check, that was intentional to allow implementations to throw an informative error about PlainMonthDay only having `monthCode` and not `month`. (Unfortunately I can't find the reference where this was discussed, at the moment)

Reply:
Has this been changed to match now? I went back to re-review it and it looks like it does match and throw that informative error.  In which case, resolved.

Reply:
I didn't change it, but it sounds like it's correct now :smile: